csharp/src/Apache.Arrow/Field.cs

```diff
@@ -13,10 +13,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Apache.Arrow.Types;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Apache.Arrow.Types;
```

Comment:
(nit) we should revert this change. It isn't worth cluttering the history of the file for sorting namespaces.